apps/products-feed/src/pages/configuration.tsx

```diff
@@ -1,24 +1,140 @@
+              read access and provide its credentials to the app.
+            </Paragraph>
+            <Paragraph size={"small"}>
+              Please use this bucket only for XMLs. They meant to be public - but other files may
+              leak if you put them to the same bucket.
+            </Paragraph>
+            <Paragraph size={"small"}>
+              Read about generating AWS credentials{" "}
+              <TextLink
+                href={
+                  "https://docs.aws.amazon.com/powershell/latest/userguide/pstools-appendix-sign-up.html"
+                }
+                newTab
+              >
+                here
+              </TextLink>
+            </Paragraph>
+          </Box>
+        }
+      />
+      <AppSection
+        __marginBottom="100px"
+        includePadding
+        heading={"Channels configuration"}
+        mainContent={
+          <Box>
+            <ChannelsConfigAccordion />
+          </Box>
+        }
+        sideContent={
+          <Box>
+            <Paragraph size={"small"}>App will generate separate feed for each channel</Paragraph>
+            <Paragraph size={"small"}>
+              Provide your storefront homepage URL and product template. Use following placeholders:
+            </Paragraph>
+            <ul>
+              <li>
+                <Text size={"small"}>
+                  <code>{"{productSlug}"}</code> - to inject slug of the product
+                </Text>
+              </li>
+              <li>
+                <Text size={"small"}>
+                  <code>{"{variantId}"}</code> - to inject id of the variant
+                </Text>
+              </li>
+              <li>
+                <Text size={"small"}>
+                  <code>{"{productId}"}</code> - to inject id of the product
+                </Text>
+              </li>
+            </ul>
+            <Paragraph size={"small"}>For example following pattern:</Paragraph>
+            <Paragraph size={"small"}>
+              <code>{"https://my-shop.com/p/{productSlug}/{variantId}"}</code>
+            </Paragraph>
+            <Paragraph size={"small"}>Will produce:</Paragraph>
+            <Paragraph size={"small"}>
+              <code>{"https://my-shop.com/p/t-shirt/Xyp2asZ"}</code>
+            </Paragraph>
+          </Box>
+        }
+      />
+      <AppSection
+        __marginBottom="100px"
+        includePadding
+        heading={"Categories mapping"}
+        mainContent={
+          <Box>
+            <CategoryMappingPreview />
+            <Box display={"flex"} justifyContent={"flex-end"}>
+              <Button marginTop={8} onClick={() => push("/categories")}>
+                Map categories
+              </Button>
+            </Box>
+          </Box>
+        }
+        sideContent={
+          <Box>
+            <Paragraph size={"small"}>
+              Map Saleor categories to pre-defined Google categories
```

Comment:
Might be a good idea to mention that it's optional and link the documentation https://support.google.com/merchants/answer/6324436?hl=en